src/elements/author-profile/src/index.tsx

```diff
@@ -112,75 +190,20 @@ const AuthorProfile: React.FC<Props> = ({
           }}
         >
           {bio}
-        </Styled.p>
-        <div
-          sx={{
-            marginTop: 'sm',
-            paddingLeft: 'xs',
-            variant: styles('details')
-          }}
-        >
-          {email && (
-            <Styled.a
-              href={'mailto:' + email}
-              sx={{
-                textDecoration: 'none',
-                cursor: 'pointer',
-                borderBottom: 0
-              }}
-            >
-              <div
-                sx={{
-                  display: 'flex'
-                }}
-              >
-                <Icon glyph="email-filled" size={22} color="grey-80" />
-                {showDetails && <span>{email}</span>}
-              </div>
-            </Styled.a>
-          )}
-
-          {twitterLink && (
-            <Styled.a
-              href={twitterLink}
-              target={socialsHrefTarget}
-              sx={{
-                textDecoration: 'none',
-                cursor: 'pointer',
-                borderBottom: 0
-              }}
-            >
-              <div
-                sx={{
-                  display: 'flex'
-                }}
-              >
-                <Icon glyph="twitter" size={22} color="grey-80" />
-                {showDetails && <span>{twitterLink}</span>}
-              </div>
-            </Styled.a>
-          )}
-          {linkedInLink && (
-            <Styled.a
-              href={linkedInLink}
-              target={socialsHrefTarget}
-              sx={{
-                textDecoration: 'none',
-                cursor: 'pointer',
-                borderBottom: 0
-              }}
-            >
-              <div
-                sx={{
-                  display: 'flex'
-                }}
-              >
-                <Icon glyph="linkedIn" size={22} color="grey-80" />
-                {showDetails && <span>{linkedInLink}</span>}
-              </div>
-            </Styled.a>
-          )}
-        </div>
+        </BioElement>
+        {socials.length === 0 ? (
```

Comment:
It is preferable to use <If condition={...}... rather than ternary expressions

See https://www.npmjs.com/package/react-if for more info